### ISSUE TITLE
WRSH Arts in WordPress

### ISSUE THREAD

**Issue by kmarjanovic** [TWUOnline/bug-tracking#653]
### Course
WRSH Arts certificate Hub & 5 courses

### Instructions
- [ ] @cmadland Please clone [FNDN 101](https://create.twu.ca/fndn101/wp-admin/) and save as a WP Course Template (like this [template](https://create.twu.ca/template/)
- [ ] Kelly will then delete content & add placeholders
- [ ] @MaraChequer  please find wrsh images to replace fndn ones
- [ ] @Nataliadsanchez please check features/content of [FNDN 101](https://create.twu.ca/fndn101/) site to see what we would like to change for wrsh courses.

Once these are done... (by next Thursday?)
- [ ] Copy new wrsh template for new sites: WRSH Arts Program Hub (see old [link](https://create.twu.ca/worshiparts/)), WRSH 239, WRSH 401, WRSH 301.
- [ ] Update [WRSH 101](https://create.twu.ca/wrsh101/home/) (Git content) in WP to new theme
- [ ] Update [WRSH 201](https://create.twu.ca/wrsh201/) in WP to new theme

**Comment by cmadland** [comment]
Happy to help with this, however, our cloning superpowers were too powerful and we had to get rid of them. Barish has been poking about with some alternate plugins, but I don't believe that is ready to go.

Workaround is to export content from a course and import into a new site...will get on that.

**Comment by cmadland** [comment]
ok, imported the site to https://create.twu.ca/wrsh-template/wp-admin

**Comment by kmarjanovic** [comment]
Thank you @cmadland !

I am a little confused though why some of the features (e.g. [Units](https://create.twu.ca/fndn101/units/)) didn't get copied over.  Did you use this [fndn ](https://create.twu.ca/fndn101/)course?  Or do some features not transfer?  e.g. drop down menu at the top, colour scheme. etc.?

**Comment by kmarjanovic** [comment]
@cmadland could you help import the Unit tab - and maybe just 1 unit so we can make a template?
https://create.twu.ca/fndn101/units/
https://create.twu.ca/fndn101/unit-1/

Also - please add the assessment section https://create.twu.ca/fndn101/assessment/
Thank you!

**Comment by cmadland** [comment]
> I am a little confused though why some of the features (e.g. [Units](https://create.twu.ca/fndn101/units/)) didn't get copied over. Did you use this [fndn ](https://create.twu.ca/fndn101/)course? Or do some features not transfer? e.g. drop down menu at the top, colour scheme. etc.?

- menu is changed now to the same for fndn101...I think that takes care of https://github.com/TWUOnline/bug-tracking/issues/653#issuecomment-1326705702

**Comment by kmarjanovic** [comment]
Perfect!  Thank you!!

**Comment by cmadland** [comment]
I presume I should hold off on creating 239, 301, 401 etc until the template site is /chef's kiss/

**Comment by kmarjanovic** [comment]
Yes, hold off for a bit - thanks.  (btw I had to google chef's kiss;)

**Comment by kmarjanovic** [comment]
@MaraChequer @Nataliadsanchez - I added the Unit 1 template to the other units, and I'm just going to play around with Unit 5 (https://create.twu.ca/wrsh-template/units/unit-5-testing-plugins/) to see what other features might work in our units.

Need to pick @cmadland 's brain about tagging & categories in WP courses...maybe Monday's CoP?

**Comment by cmadland** [comment]
I'll be 'on vacation' 'til wednesday...gotta get this paper done.

**Comment by kmarjanovic** [comment]
@cmadland @MaraChequer @Nataliadsanchez 

Reviving this issue, as we're getting ready to use the [WRSH ARTS Template](https://create.twu.ca/wrsh-template/units/unit-1/).   Colin has suggested Learning Activities be turned into posts (see [EDCI example](https://edtechuvic.ca/edci335/category/learning-motivation-and-theory/)), which would be great for student comments, and may be easier for readability.
@cmadland will you test it out on a unit?  Once it's ready, let's meet to confirm the layout.

@MaraChequer could we create a selection of worship arts graphics?  I've started one [here](https://www.notion.so/twuid/WRSH-Arts-Graphics-df3e15504bc24ac4bb066e8151e2df65) from what was in my files.  We would need (I think) 1 per unit for each of the 5 courses.  David Squires will confirm which images to use, but it would be good to show options. (e.g. see unit titles for WRSH 101, 201)

**Comment by MaraChequer** [comment]
Thanks Kelly, I have added images to the notion repository you created. Please let me know David Squires' image selection and I will prepare the files accordingly.

**Comment by kmarjanovic** [comment]
Adding @Jeff-TWU & @chloecwh  

It appears this request was made a year ago! Very excited to get this moving again!!!

**Comment by chloecwh** [comment]
Mockups we discussed today: https://www.canva.com/design/DAF1BVXL2Qo/iZwuG5H51Eg8iQzqTeSnxg/edit?utm_content=DAF1BVXL2Qo&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton

**Comment by kmarjanovic** [comment]
SAMC projects on hold as we focus on TOP.
